From c38c604b06a63f690d67318d4ef97ac556264876 Mon Sep 17 00:00:00 2001
From: Jack Hill <jackhill@jackhill.us>
Date: Thu, 18 Nov 2021 00:46:58 -0500
Subject: [PATCH] meson: remove extraneous positional argument

* data/meson.build (desktop_file)
  (metainfo_file): Remove extraneous positional argument.

Closes #186

Upstream-Status: Backport
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 data/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index fd570b3..4e2e2e8 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -1,7 +1,6 @@
 gnome = import('gnome')
 
 desktop_file = i18n.merge_file(
-  'desktop',
   type: 'desktop',
   input: 'org.gnome.Screenshot.desktop.in',
   output: 'org.gnome.Screenshot.desktop',
@@ -23,7 +22,6 @@ if desktop_file_validate.found()
 endif
 
 metainfo_file = i18n.merge_file(
-  'appdata',
   input: 'org.gnome.Screenshot.metainfo.xml.in',
   output: 'org.gnome.Screenshot.metainfo.xml',
   po_dir: join_paths(meson.current_source_dir(), '../po'),
